superset-frontend/plugins/legacy-plugin-chart-chord/src/Chord.js

```diff
@@ -130,9 +130,9 @@ function Chord(element, props) {
     .text(
       d =>
         `${nodes[d.source.index]} → ${nodes[d.target.index]}: ${f(
```

Comment:
The diff looks a bit misleading.
I simply swapped `d.source.value` and `d.target.value` (line 133 and 135)